src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -2269,4 +2269,55 @@ public Object executeScript(String script) {
         return getSeleniumHelper().executeJavascript(statement);
     }
 
+    /**
+     * @param place to check
+     * @return true if content of place is valid (according to validation pattern)
+     */
+    public boolean valueOfIsValid(String place) {
+        T element = getElementToRetrieveValue(place, null);
```

Comment:
Why not use `T element = waitUntil(d -> getElementToRetrieveValue(place, null));` to ensure waiting is done before deciding the element could not be found?
Then the if/throw is no longer needed

Reply:
Nice! I will update.